packages/server/package.json

```diff
@@ -18,7 +18,7 @@
     "test": "bash scripts/test.sh",
     "test:memory": "jest --maxWorkers=2 --logHeapUsage --forceExit",
     "test:watch": "jest --watch",
-    "build:docker": "yarn build && docker build . -t app-service --label version=$BUDIBASE_RELEASE_VERSION --build-arg BUDIBASE_VERSION=$BUDIBASE_RELEASE_VERSION",
+    "build:docker": "yarn nx build && docker buildx build ../.. -t app-service --label version=$BUDIBASE_RELEASE_VERSION --build-arg BUDIBASE_VERSION=$BUDIBASE_RELEASE_VERSION -f Dockerfile.v2 --platform linux/amd64,linux/arm64",
```

Comment:
How come the path to the build context has changed here?

Reply:
The new images (v2) are created to be run from the root of the monorepo, as these require access to other files

Reply:
Are you worried at all about the affect this will have on the cacheability of the image?

Reply:
No, we have really strict dockerignore file and the docker image is only copying what is required, instead of using wildcards. So it should not make any difference